PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### What does this PR do?
+#### Where should the reviewer start?
+#### How should this be manually tested?
```

Comment:
How can the reviewer verify this patch?

We want to avoid manually verifying patches.

Reply:
+1 when I saw this on the zoom earlier today it raised my spidey sense. I'd definitely not want to manually test a PR. In general, the PR should be "testable" just by including it and then running the standard regressions. (Especially if it adds regression tests or alters them.) I'm not familiar enough with the code base to ascertain what regr tests we have in place though.

Reply:
Yep, I agree. Going with "How can the reviewer verify this PR?"